extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -71,6 +71,13 @@ public function getTopContributorsData() {
 
 		$topContributorsDetails = $this->getContributorsDetails( $topContributors );
 
+		global $wgLang;
+		$redirect = 'redirect=' . urlencode( SpecialPage::getTitleFor( 'CommunityPage' )->getCanonicalURL() );
```

Comment:
Isn't the special page name just `Community`? Also, since you're within the special page class here, can't you just use `$this->getTitle()->getCanonicalURL()` since `$this->getTitle()` references the Community special page?